charts/geoserver/values.yaml

```diff
@@ -3,26 +3,45 @@
 # Declare variables to be passed into your templates.
 
 image:
-  repository: docker.osgeo.org/geoserver
+#  repository: docker.osgeo.org/geoserver
+  repository: ywkim312/geoserver
   pullPolicy: IfNotPresent
   # Overrides the image tag whose default is the chart appVersion.
   tag: ""
 
-username: admin
-password: geoserver
+auth:
+  username: admin
+  password: geoserver
+  # use an existing secret if specified
+  # existingSecret: geoserver-password
+  # passwordKey: geoserver-password
+
+cors:
+  enabled: true
+
+extension:
+  install: false
+  stableExtension: ""
+
+demoData:
+  skip: true
 
 startup:
   failureThreshold: 10
   periodSeconds: 30
 
 imagePullSecrets: []
+
 nameOverride: ""
 fullnameOverride: ""
 
 persistence:
   size: 10Gi
   accessModes: ReadWriteOnce
   #existingClaim: someclaim
+  existingClaim: geoserver-pvc-test
+
+whitelist: geoserver.example.com
```

Comment:
add some comment, and maybe comment this by default? or set to a blank value.

Reply:
Added comments to explain it. And values changed to empty string